src/braket/jobs/quantum_job_creation.py

```diff
@@ -147,9 +147,6 @@ def prepare_quantum_job(
 
     Returns:
         dict: Hybrid job tracking the execution on Amazon Braket.
-
-    Raises:
-        ValueError: Raises ValueError if the parameters are not valid.
     """
```

Comment:
Why removed?

Reply:
No value error is thrown here.

Reply:
No error in scope

Reply:
But what if it can be thrown from a function that it calls? Wouldn't that still be a valid "Raises"? This is actually a pretty big deal. Let's talk about it.